utils_linux.go

```diff
@@ -314,10 +314,10 @@ func validateProcessSpec(spec *specs.Process) error {
 		return errors.New("process property must not be empty")
 	}
 	if spec.Cwd == "" {
-		return errors.New("Cwd property must not be empty")
+		return errors.New("cwd property must not be empty")
```

Comment:
For others reviewing (I was wondering if this should be CWD), but the property is lowercase in JSON, so probably makes sense to match that https://github.com/opencontainers/runc/blob/e42446f7c68184fd1c1005f0e59a60312defb83a/vendor/github.com/opencontainers/runtime-spec/specs-go/config.go#L78-L80